pkg/api/types.go

```diff
@@ -14,6 +15,49 @@ const (
 	StatusError        OpenSCAPStatus = "Error"
 )
 
+// ScanResult represents the compacted result of all scans performed on the image
+type ScanResult struct {
+	// ImageName is a full pull spec of the input image
+	ImageName string `json:"imageName"`
+	// ImageIUD is a SHA256 identifier of the scanned image
+	ImageID string `json:"imageID"`
+	// Results contains compacted results of various scans performed on the image.
+	// Empty results means no problems were found with the given image.
+	Results []Result `json:"results,omitempty"`
+}
+
+// Result represents the compacted result of a single scan
+type Result struct {
+	// Name is the name of the scanner that produced this result
+	Name string `json:"name"`
+	// ScannerVersion is the scanner version
+	ScannerVersion string `json:"scannerVersion"`
+	// Timestamp is the exact time this scan was performed
+	Timestamp time.Time `json:"timestamp"`
+	// Reference contains URL to more details about the given result
+	Reference string `json:"reference"`
+	// Description describes the result in human readable form
+	Description string `json:"description,omitempty"`
+	// Summary contains a list of severities for the given result
+	Summary []Summary `json:"summary,omitempty"`
+}
+
+type Severity string
+
+var (
+	LowSeverity       Severity = "low"
```

Comment:
switch the name so that type is first (`SeverityLow`) so it shows nicely in code complete :-)